src/styled/ConvertCard.ts

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+const ConvertCard = styled.div`
```

Comment:
Can we use a named export here? You can read about it here: https://basarat.gitbook.io/typescript/main-1/defaultisbad

Reply:
sure thing!